user-interface/src/main/java/life/qbic/datamanager/views/general/contact/AutocompleteContactField.java

```diff
@@ -143,6 +143,7 @@ public void setContact(Contact contact) {
 
   public void setItems(List<Contact> contacts) {
     contactSelection.setItems(contacts);
+    contactSelection.setVisible(!contacts.isEmpty());
   }
```

Comment:
I'd move this logic into the catch block of the access denied exception if possible. Our fields should not decide on their visibility but the layout containing the field should imo 🤔